### ISSUE TITLE
attachment is not working

### ISSUE THREAD

**Issue by ryoppippi** [sveltejs/eslint-plugin-svelte#1219]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.

### What version of ESLint are you using?

^9.13.0

### What version of `eslint-plugin-svelte` are you using?

^3.6.0

### What did you do?

```svelte
<script lang="ts">
	import type { Attachment } from 'svelte/attachments';


	const myAttachment: Attachment = (element) => {
		console.log(element.nodeName); // 'DIV'

		return () => {
			console.log('cleaning up');
		};
	};
</script>

<div {@attach myAttachment}>...</div>

```

### What did you expect to happen?

linting without error

### What actually happened?

got the error like this
```
/Users/ryoppippi/ghq/github.com/vim-jp-radio/LP/src/lib/Background/Background.svelte
  54:2  error  Parsing error: Unknown directive or attribute (AttachTag) are not supported
```

### Link to **GitHub Repo** with Minimal Reproducible Example

Sorry no reproduction repo.
I think we need to update parser

**Comment by baseballyama** [comment]
Already we supports the `{@attach}`.
Please upgrade [svelte-eslint-parser](https://github.com/sveltejs/svelte-eslint-parser) v1.2.0.

**Comment by ryoppippi** [comment]
@baseballyama Thanks!! it works
